client/src/components/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Card, Box, Typography, TextField, Button, Alert } from '@mui/material';
+import axios from 'axios';
+import routes from '../../routes';
+
+
+const SignIn = () => {
+  const navigate = useNavigate();
+  const [password, setPassword] = useState('');
+  const [email, setEmail] = useState('');
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const handleSignIn = async (e) => {
+    e.preventDefault();
+    const userData = {
+      email,
+      password
+    };
+
+    try {
+      await axios.post('/api/auth/sign-in', userData, {
+        headers: { 'Content-Type': 'application/json' },
+        withCredentials: true
+      })
+      navigate(routes.findDrug)
+
+    } catch (err) {
+      if (err?.response) setErrorMessage(<Alert variant="outlined" severity="error">{err?.response?.data?.message}</Alert>)
+    }
+  }
+
+  const navigateToSignUp = () => {
+    navigate(routes.signUp)
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        <Card
+          elevation={16}
+          sx={{
+            flexBasis: 350,
+            mx: 2,
+            my: 3,
+            p: 2,
+            display: 'flex',
+            flexDirection: 'column',
+            boxShadow: 'none'
+          }}
+        >
+          <Box sx={{ alignSelf: 'start', display: 'flex', alignItems: 'end', mb: 3 }}>
+            <img src="/180.png" alt="icon" height="35px" loading="lazy" />
+            <Typography>Medical Calculator</Typography>
+          </Box>
+          <Typography variant="h4" component="div">
+            Logowanie
+          </Typography>
+          <Box>
+            {errorMessage}
```

Comment:
In Sign-up we display this error differently I would suggest we keep it consistent. We can reuse the component SignUpError (probably we should rename it to FormError or sth like this) and use it here as well.

Message itself could be stored as text instead of the component and be passed down as prop